etna/libs/sklearn/metrics.py

```diff
@@ -0,0 +1,117 @@
+"""
```

Comment:
I'm not sure if we should add this into `etna.libs`, but it seems like change a very small and we should give a link to a license of sklearn somewhere.